fe/fe-core/src/main/java/org/apache/doris/analysis/InsertStmt.java

```diff
@@ -604,9 +604,7 @@ private void analyzeRow(Analyzer analyzer, List<Column> targetColumns, List<Arra
             extentedRow.addAll(row);
             
             for (Pair<Integer, Column> entry : origColIdxsForExtendCols) {
-                if (entry == null) {
```

Comment:
I don't know whether it's right solution. But this problem didn't happen past, so I just remove it.

Reply:
indeed this is a wrong logic